docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -106,7 +106,7 @@ In this example, you connect a data source (a data generator) that creates Kafka
 
 ifndef::qs[]
 .Example flow of messages from a data source to a data sink
-image::{imagesdir}/connectors-getting-started-connectors/connectors-example-diagram.png[Image of data flowing from a data source to a data sink]
+image::connectors-getting-started-connectors/connectors-example-diagram.png[Image of data flowing from a data source to a data sink]
```

Comment:
Might want to check the file name and location? This is what I see when I check your branch?

```suggestion
image::connectors-ex-diagram.png[Image of data flowing from a data source to a data sink]
```

Reply:
It is correct for downstream. I removed the upstream image file